content/en/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -117,16 +117,14 @@ if the `kubeadm init` command was called with `--upload-certs`.
  - The CA hash is not normally known until the control-plane node has been provisioned,
    which can make it more difficult to build automated provisioning tools that
    use kubeadm. By generating your CA in beforehand, you may workaround this
-   limitation though.
+   limitation.
 
 #### Token-based discovery without CA pinning
 
-_This was the default in Kubernetes 1.7 and earlier_, but comes with some
-important caveats. This mode relies only on the symmetric token to sign
+This mode relies only on the symmetric token to sign
 (HMAC-SHA256) the discovery information that establishes the root of trust for
-the control-plane. It's still possible in Kubernetes 1.8 and above using the
-`--discovery-token-unsafe-skip-ca-verification` flag, but you should consider
-using one of the other modes if possible.
+the control-plane. To use the mode the joining nodes must skip the hash validation of the
+CA public key, using `--discovery-token-unsafe-skip-ca-verification`.
```

Comment:
I think we should preserve the recommendation ", but you should consider using one of the other modes if possible."